src/roboteam_ai.cpp

```diff
@@ -77,7 +78,10 @@ int main(int argc, char *argv[]) {
     // initialize the interface
     QApplication a(argc, argv);
     setDarkTheme();
-    window = std::make_shared<ui::MainWindow>();
+
+    // Todo make this a not-global-static thingy
+    rtt::world_new::World* worldManager = rtt::world_new::World::instance();
+    window = std::make_shared<ui::MainWindow>(*worldManager);
     window->setWindowState(Qt::WindowMaximized);
```

Comment:
Can't you const ref from application manager? Application manager should also own an instance. Having 2 instances is quite ugly...

Reply:
Maybe I am understanding it wrong here though.

Reply:
I'm getting the global instance here. There is always exactly one instance. Always.